src/style.css

```
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--5 > .stars__star {
+  background-image: url(images/star-active.svg);
+}
+
+.stars .stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:nth-last-child(n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-last-child(n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-last-child(n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-last-child(n+2) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--5 > .stars__star {
  background-image: url(images/star-active.svg);
}

.stars .stars__star:last-child {
  margin-right: 0;
}

.stars--1 > .stars__star:nth-last-child(n+5)
.stars--2 > .stars__star:nth-last-child(n+4) 
.stars--3 > .stars__star:nth-last-child(n+3)
.stars--4 > .stars__star:nth-last-child(n+2) {
  background-image: url(images/star-active.svg);
}
```

давай скомбінуємо селектори з однаковивими стилями